install-paperless-ngx.sh

```diff
@@ -329,16 +329,29 @@ sed -i "s#- \./consume:/usr/src/paperless/consume#- $CONSUME_FOLDER:/usr/src/pap
 
 if [[ -n $MEDIA_FOLDER ]] ; then
 	sed -i "s#- media:/usr/src/paperless/media#- $MEDIA_FOLDER:/usr/src/paperless/media#g" docker-compose.yml
+	sed -i "/^\s*media:/d" docker-compose.yml
 fi
 
 if [[ -n $DATA_FOLDER ]] ; then
 	sed -i "s#- data:/usr/src/paperless/data#- $DATA_FOLDER:/usr/src/paperless/data#g" docker-compose.yml
+    sed -i "/^\s*data:/d" docker-compose.yml
 fi
 
 if [[ -n $POSTGRES_FOLDER ]] ; then
 	sed -i "s#- pgdata:/var/lib/postgresql/data#- $POSTGRES_FOLDER:/var/lib/postgresql/data#g" docker-compose.yml
+    sed -i "/^\s*pgdata:/d" docker-compose.yml
 fi
 
+# remove trailing blank lines from end of file
+sed -i -e :a -e '/^\n*$/{$d;N;};/\n$/ba' docker-compose.yml
+# if last line in file contains "volumes:", remove that line since no more named volumes are left
```

Comment:
Is this still true since #132? I don't think redisdata ever gets removed from `volumes:`.

Reply:
That's true, but until 1.6.0 is released the script uses the composer files from the main branch and there we don't have the redis volume

Reply:
We could simply wait merging this one until 1.6.0 is released. In that case I would remove the coding that deletes the dangling volume statement. I didn't anticipate 1.6.0 being ready so quickly. Just let me know what you prefer.

Reply:
I see, that makes sense.

Okay, waiting until merging might be best rather than returning later to fix. What do you think?

Reply:
Agreed, I will update the PR tomorrow and remove the corresponding statements